### PR TITLE
Mongodbroute

### PR DESCRIPTION
**Pull request by SvLowman**
Connect with MongoDB and get data from there that can be displayed in the front end. Still throws a warning in GalleryPage.js, line 16.

### CODE REVIEW

**Review by domko95 — APPROVED** [review]
🚀 LGTM 🚀

File: client/src/pages/GalleryPage.js [hunk 1]
```diff
@@ -1,20 +1,40 @@
...
-      <h2>Das hier ist die Album-Seite 🤩</h2>
-    </section>
-    <Display>
-      <img src="" alt="testImage" />
-      <p>Display für Response der GET-Route</p>
-    </Display>
-  </div>
-);
+const GalleryPage = () => {
+  const { data, loading, error, doFetch } = useAsync(() => getData("2"));
```
Comment (phlgr) [inline client/src/pages/GalleryPage.js:12]: Is this for testing?

File: client/src/utils/api.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const getData = async (dataTag) => {
+  const response = await fetch(`/api/sven/${dataTag}`);
```
Comment (phlgr) [inline client/src/utils/api.js:2]: You wouldn't have a custom API Endpoint for every user
```suggestion
  const response = await fetch(`/api/users/${user}/${dataTag}`);
```

File: lib/data.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const { collection } = require("./database");
+
+async function getData(dataTag) {
+  const imgEntry = await collection("sven")
```
Comment (phlgr) [inline lib/data.js:4]: You don't need a collection for every user, just make a general user collection

File: server.js [hunk 4]
```diff
@@ -23,6 +28,22 @@ app.post("/api/upload", async (request, response) => {
   }
 });
 
+// GET-Route from MongoDB
+app.get("/api/sven/:tags", async (request, response) => {
+  const { tags } = request.params;
+  try {
+    const entryValue = await getData(tags);
```
Comment (phlgr) [inline server.js:35]:
```suggestion
    const entryValue = await getImage(tags);
```
What kind of data do you get

**Review by phlgr — CHANGES_REQUESTED** [review]

File: client/src/pages/GalleryPage.js (hunk 1 above)
Comment (SvLowman) [reply client/src/pages/GalleryPage.js:12]: Yes, the argument "2" is just to see if it works.

File: client/src/utils/api.js (hunk 2 above)
Comment (SvLowman) [reply client/src/utils/api.js]: 👍

File: server.js (hunk 4 above)
Comment (SvLowman) [reply server.js]: 👍
Thanks for reviewing!

**Pull request merged**